File: src/main/webapp/asset/js/BMIService.js
var BMIService = {

	getIndex: function(weight, height, callback) {
		$.ajax({
		  url: 'BMIController',
		  data: {'weight': weight, 'height': height},
		  success: function(result) {
		  	callback(parseFloat(result));
		  },
		  error: function() {
		  	callback(null);
		  }
		});
	},
	
	getDescription: function (index) {
		//Fazer em casa
		return "";
	}
	
};

File: src/main/java/br/edu/univas/si/lab6/bmi/service/BMIService.java
package br.edu.univas.si.lab6.bmi.service;

public class BMIService {
	
	private BMIService(){
		//Jamais será instanciado com o new
	}

	public static Float getIndex(Float weight, Float height) {
		return weight / (height * height);
	}
	
	public static String getDescription (Float bmiIndex) {
		//Fazer isso em casa
		return "";
	}
}
